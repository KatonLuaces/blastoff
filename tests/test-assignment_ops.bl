A = 1;
print(toString(A));
A*=2;
print(toString(A));
A~=2;
print(toString(A));
A@=2;
print(toString(A));
A+=1;
print(toString(A));
A^=2;
print(toString(A));
A:=A;
print(toString(A));
