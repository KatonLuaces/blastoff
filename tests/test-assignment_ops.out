1
2
4
8
9
81
81
81
